### DISCUSSION TITLE
Is DataTriggerBehavior supported in Uno Behavior SDK

### DISCUSSION THREAD

**Discussion by cmonto** [unoplatform/uno#12897] [Q&A]
Used `Uno.Microsoft.Xaml.Behaviors.WinUI.Managed` and added a `DataTriggerBehavior` to call a Storyboard. This seem to be not getting triggered at all in Android, but works fine in Windows. Can someone pls confirm this works or suggest a xaml based workaround?
`<i:Interaction.Behaviors>
                            <ic:DataTriggerBehavior Binding="{Binding ThumbnailImageSource,Converter={StaticResource IsNotNullConverter}}" Value="true">
                                <im:ControlStoryboardAction Storyboard="{StaticResource ListThumbnailAnimation}" />
                            </ic:DataTriggerBehavior>
                        </i:Interaction.Behaviors>`

**Comment by jeromelaban** [comment]
Thanks for the report. Can you provide a repro sample of your problem? We'll be sure to look at the same behavior.

**Comment by Hurricane31337** [comment]
This is not a sample project but here is the documentation:
https://github.com/Microsoft/XamlBehaviors/wiki/DataTriggerBehavior

In essence, it would be really handy to be able to write this:
```xaml annotate
<Rectangle x:Name="DataTriggerRectangle">
    <Interactivity:Interaction.Behaviors>
        <Interactions:DataTriggerBehavior Binding="{Binding Value, ElementName=slider}" ComparisonCondition="GreaterThan" Value="50">
            <Interactions:ChangePropertyAction TargetObject="{Binding ElementName=DataTriggerRectangle}" PropertyName="Fill" Value="{StaticResource PaleYellowBrush}"/>
        </Interactions:DataTriggerBehavior>
        <Interactions:DataTriggerBehavior Binding="{Binding Value, ElementName=slider}" ComparisonCondition="LessThanOrEqual" Value="50">
            <Interactions:ChangePropertyAction TargetObject="{Binding ElementName=DataTriggerRectangle}" PropertyName="Fill" Value="{StaticResource RoyalBlueBrush}"/>
        </Interactions:DataTriggerBehavior>
    </Interactivity:Interaction.Behaviors>
</Rectangle>
```
